website/docs/cloud-docs/api-docs/projects.mdx

```diff
@@ -485,7 +485,7 @@ $ curl \
 
 ## Delete a project
 
-A project cannot be deleted if it contains workspaces.
+A project cannot be deleted if it contains stacks or workspaces.
```

Comment:
Stacks is a beta feature and only available in cloud, so is this still ok? Do we need to let the user know about the beta here?

Reply:
I feel like it's ok as is but thats a good callout - im not actually sure 😅

Reply:
I'll see what the Documentation team says!